examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<p>The wave disapear in the middle of the simulation, is there a stability issue there or is it just becoming zeros?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I think it not capturing full timesteps in the video?Atila could you double check?</p><p>It seems like at the last frames a new blue trace is starting at the center</p>

Reply:
<p>Thx for the review! I actually pondered about that, I'm going to have to investigate, but indeed the video is capturing what is present in the etasave function, so I might have to review how it is capturing snapshots.</p>